Patch kindly borrowed from Mageia.

* asturm@gentoo.org: Dropped unnecessary and backwards incompatible
xmp_exiv2.hpp include.

https://gitlab.gnome.org/GNOME/gthumb/issues/30

diff -Nru a/extensions/exiv2_tools/exiv2-utils.cpp b/extensions/exiv2_tools/exiv2-utils.cpp
--- a/extensions/exiv2_tools/exiv2-utils.cpp	2018-06-17 08:24:44.000000000 +0200
+++ b/extensions/exiv2_tools/exiv2-utils.cpp	2018-12-31 15:51:50.912329232 +0100
@@ -32,7 +32,7 @@
 #include <sstream>
 #include <vector>
 #include <iomanip>
-#include <exiv2/xmp.hpp>
+#include <exiv2/exiv2.hpp>
 #include <gthumb.h>
 #include "exiv2-utils.h"
 
@@ -1073,7 +1073,11 @@
 
 	try {
 		if (exifData.empty()) {
+#if EXIV2_TEST_VERSION(0,27,0)
+			throw Exiv2::Error(Exiv2::kerErrorMessage, " No Exif data found in the file");
+#else
 			throw Exiv2::Error(1, " No Exif data found in the file");
+#endif
 		}
 		Exiv2::ExifData::const_iterator end = exifData.end();
 		for (Exiv2::ExifData::const_iterator i = exifData.begin(); i != end; ++i) {
